### PR TITLE
Update gold sponsors with Jolicode

### PR DESCRIPTION
**Pull request by Yago**
@WengerK When you're ready 😉

### CODE REVIEW

File: src/components/Sponsors/Sponsors.tsx [hunk 1]
```diff
@@ -45,7 +31,7 @@ const Sponsors = (): JSX.Element => {
               height="54"
             />
           </a>
-          {/* <a
+          <a
             href="https://jolicode.com/"
             target="_blank"
             rel="noreferrer"
```
Comment (WengerK) [inline src/components/Sponsors/Sponsors.tsx:37]:
```suggestion
```

File: src/components/Sponsors/Sponsors.tsx [hunk 2]
```diff
@@ -45,7 +31,7 @@ const Sponsors = (): JSX.Element => {
               height="54"
             />
           </a>
-          {/* <a
+          <a
             href="https://jolicode.com/"
```
Comment (WengerK) [inline src/components/Sponsors/Sponsors.tsx:35]:
```suggestion
            href="https://jolicode.ch/"
```

File: src/components/Sponsors/Sponsors.tsx [hunk 3]
```diff
@@ -17,20 +17,6 @@ const Sponsors = (): JSX.Element => {
...
-              src="/vectors/antistatique.svg"
-              alt="Antistatique SA"
-              tw="w-auto"
-              width="235"
-              height="42"
-            />
-          </a>
           <a
             href="https://liip.ch/"
             target="_blank"
```
Comment (WengerK) [inline src/components/Sponsors/Sponsors.tsx:22]: msut remove no-referrer here (liip)

**Review by WengerK — COMMENTED** [review]
we should keep referrer for our sponsors so they know how many people come from Webmardi :)

File: src/components/Sponsors/Sponsors.tsx (hunk 1 above)
Comment (Yago) [reply src/components/Sponsors/Sponsors.tsx:37]: And like that ?
```suggestion
            rel="noopener"
```

File: src/components/Sponsors/Sponsors.tsx (hunk 3 above)
Comment (Yago) [reply src/components/Sponsors/Sponsors.tsx:22]: I will already do that in prod 😉

**Pull request merged**